src/live.rs

```diff
@@ -158,6 +162,163 @@ pub fn pxe_ignition_unwrap(config: &PxeIgnitionUnwrapConfig) -> Result<()> {
     Ok(())
 }
 
+pub fn iso_kargs_modify(config: &IsoKargsModifyConfig) -> Result<()> {
+    let mut holder = CopiedFileHolder::new(&config.input, config.output.as_ref())?;
+    let (area_size, _, area_offsets) = find_karg_embed_areas(&mut holder.file)?;
+    let current_kargs = get_current_kargs(&mut holder, area_size, &area_offsets)?;
+    let new_kargs = modify_kargs(
+        &current_kargs,
+        &config.append,
+        &config.replace,
+        &config.delete,
+    )?;
+    write_iso_kargs(&mut holder, area_size, &area_offsets, &new_kargs)
+}
+
+pub fn iso_kargs_reset(config: &IsoKargsResetConfig) -> Result<()> {
+    let mut holder = CopiedFileHolder::new(&config.input, config.output.as_ref())?;
+    let (area_size, default_kargs_offset, area_offsets) = find_karg_embed_areas(&mut holder.file)?;
+    let default_kargs = get_kargs_at_offset(&mut holder, area_size, default_kargs_offset)?;
+    write_iso_kargs(&mut holder, area_size, &area_offsets, &default_kargs)
+}
+
+pub fn iso_kargs_show(config: &IsoKargsShowConfig) -> Result<()> {
+    let mut holder = CopiedFileHolder::new(&config.input, None)?;
```

Comment:
If the helper functions take a file instead of a holder, we no longer need to create a holder here.